README.md

```diff
@@ -1,117 +1,53 @@
 # profile_gpu
 
-Welcome to your new module. A short overview of the generated parts can be found
-in the [PDK documentation][1].
+![pdk-validate](https://github.com/ncsa/puppet-profile_gpu/workflows/pdk-validate/badge.svg)
+![yamllint](https://github.com/ncsa/puppet-profile_gpu/workflows/yamllint/badge.svg)
 
-The README template below provides a starting point with details about what
-information to include in your README.
+NCSA Configuration specific to Hosts with GPUs
```

Comment:
It would be good to call out that this currently only applies to NVIDIA GPUs.

Long term it might make sense to have a parameter or fact that specifies what type of GPU is installed and do things accordingly. I assume more than one brand/type of GPU is theoretically possible to be installed in the same node. 😞

Reply:
Yeah good catch. I've updated the comments to mention the DCGM parts should be turned off if you don't have an NVIDIA GPU. And opened an issue and ticket for it #2